crs_application/src/task_managers/motion_planning_manager.cpp

```diff
@@ -33,8 +33,23 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <tf2_eigen/tf2_eigen.h>
+#include <boost/optional.hpp>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <trajectory_msgs/msg/joint_trajectory.hpp>
 #include "crs_application/task_managers/motion_planning_manager.h"
 
+static const double WAIT_SERVICE_DURATION = 2.0; // secs
+static const double WAIT_SERVICE_COMPLETION_PERIOD = 30.0; // secs
+static const std::string CALL_FREESPACE_MOTION_SERVICE = "plan_free_space";
```

Comment:
This freespace motion service name should match the one in the freespace planning service node